src/services/Fedora.ts

```diff
@@ -1,24 +1,41 @@
 const http = require("needle");
 
+interface Attributes {
+    [key: string]: string;
+}
+
+interface DC_Child {
+    name: string,
+    value: string,
+    attributes: Attributes,
+    children: Array<DC_Child>,
+}
+
+interface DC {
+    name: string,
+    value: string,
+    attributes: Attributes,
+    children: Array<DC_Child>
+}
+
 class Fedora {
     baseUrl: string;
 
     constructor() {
         this.baseUrl = "http://vua6743.villanova.edu:8089/rest";
     }
 
-    async getDC(pid) {
+    async getDC(pid): Promise<DC> {
```

Comment:
If you're thinking ahead, maybe this should become a getDatastream(pid, stream) method, since I suspect we'll be pulling a number of data streams that follow this exact pattern, except use some name other than "DC" at the end.

Reply:
Of course, that causes problems for the return type here... but there could still be a getDC() method that wraps around a more loosely-typed getDatastream() if necessary.

Reply:
I agree that this needs to be abstracted a bit (since we have to auth all the requests) and getDatastream seems like a good way to do that. I guess it could return a JSON or XML?

Reply:
Yeah, that would be a good place to start. Eventually there _might_ be a need to deal with binary and RDF datastreams as well, but we can cross those bridges when we come to them.